win64.mak

```diff
@@ -8,7 +8,7 @@ SDKDIR=\Program Files (x86)\Microsoft SDKs\Windows\v7.0A
 DMD_DIR=..\dmd
 BUILD=release
 OS=windows
-DMD=$(DMD_DIR)\generated\$(OS)\$(BUILD)\$(MODEL)\dmd
+DMD="$(DMD_DIR)\generated\$(OS)\$(BUILD)\$(MODEL)\dmd"
```

Comment:
I think we should never quote when assigning to a variable, see also https://github.com/dlang/druntime/pull/2438

That also avoids the horrible quoting when invoking a sub make command, e.g. https://github.com/dlang/druntime/pull/2499/files#diff-e35cbd05831954a48feae8130a5df3f4R89 (even broken in this case).